packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -81,13 +81,24 @@ export async function handleSending(
     finalDebitAmount: maxDebitAmount,
     finalReceiveAmount: maxReceiveAmount
   })
-  deps.telemetry
-    ?.getOrCreateMetric('transactions_total', {
-      description: 'Count of funded transactions'
-    })
-    .add(1, {
-      source: deps.telemetry.getInstanceName()
-    })
+
+  if (deps.telemetry) {
+    deps.telemetry
+      .getOrCreateMetric('transactions_total', {
+        description: 'Count of funded transactions'
+      })
+      .add(1, {
+        source: deps.telemetry.getInstanceName()
+      })
+
+    deps.telemetry
+      .getOrCreateHistogramMetric('outgoing_payment_completion_time_ms', {
```

Comment:
If we go with tracking just the ILP payment, maybe we can rename this to be something ILP related? e.g. `ilp_pay_time_ms`